commercetools-models/src/main/java/io/sphere/sdk/products/search/PriceSelectionBuilder.java

```diff
@@ -21,6 +22,10 @@
     private String priceCustomerGroup;
     @Nullable
     private String priceChannel;
+    @Nullable
+    private String storeProjection;
+    @Nullable
+    private List <String> localeProjection;
```

Comment:
Here we should initialize the list, otherwise calling `plusLocaleProjection` will fail with a `NullPointerException.`:
```java
private List <String> localeProjection = new ArrayList<>();
```